src/apps/content-editor/src/app/views/ItemEdit/ItemEdit.js

```diff
@@ -176,8 +176,13 @@ export default connect((state, props) => {
         .dispatch(fetchItem(modelZUID, itemZUID))
         // select lang based on content lang
         .then(res => {
-          if (res.status === 404) {
-            throw new Error(res.message);
+          if (res.status >= 400 && res.status <= 499) {
+            this.props.dispatch(
+              notify({
+                message: `Not Found: ${res.error}`,
```

Comment:
Only a 404 response would be not found. Please tailor the messages to specific http response codes, if appropriate.

Reply:
Swapped out message
<img width="575" alt="Screen Shot 2021-06-07 at 10 30 37 AM" src="https://user-images.githubusercontent.com/22800749/121063691-7ef7b580-c77b-11eb-8d1f-fdbc50ba1354.png">

Reply:
This logic is far to encompassing. ` if (res.status >= 400 && res.status <= 499)`

- 400
- 401
- 403
- 404
- 413

Are all vastly different errors that our app can experience. The need to be handled separately, not with one encompassing error.